client/tailwind.config.js

```diff
@@ -16,9 +16,13 @@ module.exports = {
         lightGray: '#E0E5ED',
         lightGray1: '#BABABA',
         black1: '#212529',
+        gray1: '#F7F7F7',
+        gray2:'#17282680',
+        black2: '#2F302C',
```

Comment:
```suggestion
```

This one already exist as `textGray` down below, please utilize the existing one instead